cli/src/action/playlist.rs

```diff
@@ -235,7 +235,7 @@ impl Action for SubmitPlaylistAction {
             .map_err(|_| CliError::ActionError("Unable to parse provided update time".into()))?;
 
         let target_vec: Vec<String> = target.split(';').map(String::from).collect();
-
+        let rate = std::time::Duration::from_secs(1) / rate;
```

Comment:
should this read 'let time_to_wait = ...'?

Reply:
After much git ado, updated to time_to_wait